src/wizard/workflow.scss

```diff
@@ -2,49 +2,37 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
-@mixin iui-wizards-workflow {
-  @include iui-wizards;
-
-  > .iui-wizards-wrapper {
+@mixin iui-wizard-workflow {
+  ol {
     align-items: center;
+  }
 
-    > .iui-wizards-step {
-      width: auto;
-
-      > .iui-wizards-step-circle {
-        padding-left: $iui-m;
-        padding-right: $iui-m;
-        border-radius: $iui-sm;
-        width: auto;
-        height: auto;
-        min-height: $iui-l;
-        @include themed {
-          border-color: t(iui-color-foreground-positive);
-          color: t(iui-color-foreground-positive);
-        }
-      }
-
-      &.iui-wizards-step-current > .iui-wizards-step-circle {
-        border-radius: 0;
-        font-weight: $iui-font-weight-normal;
-        @include themed {
-          color: t(iui-color-foreground-accessory);
-          border-color: t(iui-color-background-positive);
-        }
-      }
+  .iui-wizard-circle {
+    padding-left: $iui-m;
+    padding-right: $iui-m;
+    border-radius: $iui-sm;
+    word-break: break-word;
+    height: 100%;
+    width: auto;
+    @include themed {
+      border-color: t(iui-color-foreground-positive);
+      color: t(iui-color-foreground-positive);
     }
+  }
 
-    > .iui-wizards-step-track,
-    > .iui-wizards-step > .iui-wizards-step-track {
-      &.iui-wizards-step-track-before,
-      &.iui-wizards-step-track-after,
-      &.iui-wizards-step-track-main {
-        top: 0;
+  .iui-wizard-step.iui-current .iui-wizard-circle {
+    border-radius: 0;
```

Comment:
I know this is not changing from how it was before but... Are we sure about this?

Sharp corners and rounded corners in different components was already bad, but now we have them in the same component 🤢

Reply:
The shapes are following [industry standard designs](https://www.gliffy.com/blog/guide-to-flowchart-symbols).

Reply:
Hm, but these are flow chart standards and for me wizard is not the same as a flowchart. Flowchart is more complex. And even if we want to follow this standard, we also have bad implementation, as oval steps are only for beginning and ending, and we are using them for not current steps.
![image](https://user-images.githubusercontent.com/36186912/129155108-da3d4ce9-269f-42b0-9aba-ee28d4b07037.png)

Reply:
> oval steps are only for beginning and ending, and we are using them for not current steps.

This was fixed in latest. Oval is only used for start/end. Everything else gets green rectangle. "current" is ignored.